src/IW/Core/Repo.hs

```diff
@@ -0,0 +1,9 @@
+{-# LANGUAGE DeriveAnyClass #-}
+
+module IW.Core.Repo
+       ( Repo (..)
+       ) where
+        
+
+-- | Data type representing a GitHub repository.
+data Repo = Repo
```

Comment:
This data type doesn't look right. Probably if it should only be used as a phantom param then it should not have any constructors at all?

Reply:
Yes, thank you!